src/parse.c

```diff
@@ -4440,7 +4440,7 @@ void Parser::checkCstyleTypeSyntax(Loc loc, Type *t, int alt, Identifier *ident)
 
 Statement *Parser::parseStatement(int flags, const utf8_t** endPtr, Loc *pEndloc)
 {
-    Statement *s;
+    Statement *s = NULL;
```

Comment:
Yikes, yes that's a bug, and I think I introduced it.  Those errors where `version = xxx` and `debug = xxx` should probably `goto Lerror` instead of breaking.  Will that be enough to silence the warning?